internal/providers/terraform/azure/util.go

```diff
@@ -114,3 +114,12 @@ func locationNameMapping(l string) string {
 func intPtr(i int64) *int64 {
 	return &i
 }
+
+func doesStrSliceContains(a []string, x string) bool {
```

Comment:
We use `contains` in other packages for this function. I don't think this is a major issue, just might want to keep it consistent in case we refactor all occurrences into a shared function.

Reply:
Yeah, I'm not sure but I thought refactoring them to a name with more details would be nice. What you think?

Reply:
Agree, I think we definitely should refactor this. And now that we're on 1.18 maybe we use a lib that does this with generics. What do you think?

I think we should handle it in a separate PR though. My thinking was just to keep the names consistent just now so it's easier to find all the cases when we refactor it. I don't have a strong opinion on what that consistent name should be.